### PR TITLE
UI fixes

### PR DESCRIPTION
**Pull request by danielreti**
Links open in new tab
Don't show blank template when project is not selected
Render files properly

### CODE REVIEW

File: web/src/project/ProjectSummary.tsx [hunk 1]
```diff
@@ -348,7 +348,11 @@ export const ProjectSummary = () => {
                             if (b[0] === "no-batch") {
                                 return -1;
                             }
-                            return a[0] > b[0] ? 1 : -1;
+                            if (isNaN(+a[0]) && isNaN(+b[0])) {
+                                //if both numbers
+                                return +a[0] - +b[0];
```
Comment (illusional) [inline web/src/project/ProjectSummary.tsx:353]: I don't understand this, are you using the `+` operator to coerce strings to integers to check if they're numbers. Is there a cleaner way, if not could you clarify why you do it this way in a comment?

File: web/src/project/ProjectSummary.tsx [hunk 2]
```diff
@@ -441,8 +447,14 @@ export const ProjectSummary = () => {
             return <></>;
         }
         return Object.entries(summary.seqr_links).map(([key, value]) => (
```
Comment (illusional) [inline web/src/project/ProjectSummary.tsx:449]: Can you add a title above the seqr buttons?

**Review by illusional — APPROVED** [review]
Looks good, two comments to address before merging (but no need for another review).

File: web/src/project/ProjectSummary.tsx (hunk 1 above)
Comment (danielreti) [reply web/src/project/ProjectSummary.tsx:353]: Fixed :)

File: web/src/project/ProjectSummary.tsx (hunk 2 above)
Comment (danielreti) [reply web/src/project/ProjectSummary.tsx:449]: Donezo

**Comment by danielreti** [comment]
Fixes #349 @vivbak FYI @illusional

**Comment by vivbak** [comment]
Amazing thanks @danielreti!

**Pull request merged**